index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <script
+      src="https://kit.fontawesome.com/f0e9415ee2.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>William Raga</title>
+  </head>
+  <body>
+    <header>
+      <h4 class="white text-normal align-center">William Raga</h4>
```

Comment:
- [ ]  You logo should be at the left side and has a link (`a` tag), Please make your logo a link